src/slicegpt/adapters/sliced_phi.py

```diff
@@ -0,0 +1,85 @@
+import torch
+import torch.nn as nn
+from transformers.models.phi.modeling_phi import PhiConfig, PhiForCausalLM, PhiModel
+
+from slicegpt.adapters.phi2_adapter import CompressedPhiDecoderLayer, Phi2ModelAdapter
+from slicegpt.modules import RMSN
+from slicegpt.rotate import slice_rotated_model
+from slicegpt.slicing_scheduler import SlicingScheduler
+
+
+class SlicedPhi2Config(PhiConfig):
+    model_type = "sliced_phi2"
+    is_composition = True
+
+    def __init__(self, sparsity=0.1, new_hidden_size=1024, **kwargs):
```

Comment:
Shall we leave the arguments `sparsity` and `new_hidden_size` uninitialised? Otherwise it could create confusion as to where the value came from when not explicitly set. And maybe we want these to be mandatory arguments?